src/components/MerchantPayout/MerchantPayoutScreen.tsx

```diff
@@ -8,7 +8,6 @@ import React, {
 import {
```

Comment:
In `InvalidCard.tsx` there is a "Continue scanning" that we missed out

Reply:
Good catch!